src/accounts-service-access.vala

```diff
@@ -222,10 +224,15 @@ public class AccountsServiceAccess : Object
         } catch (GLib.Error e) {
             warning ("unable to sync volume %f to AccountsService: %s", volume, e.message);
         }
+#else
+        return;
+#endif
+
     }
 
     private async void sync_mute_to_accountsservice (bool mute)
     {
+#if HAS_LOMIRI_SOUND_SCHEMA
```

Comment:
ifdef HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS

Reply:
Ignoring, as we agreed to simplify this in the next PR.